src/jvmTest/kotlin/io/ktor/io/JvmBufferTest.kt

```diff
@@ -0,0 +1,7 @@
+package io.ktor.io
+
+import io.ktor.io.*
+
+class JvmBufferTest : BufferTest() {
```

Comment:
let's add test with different pools and with custom pools

Reply:
Could you give me more context here? I'm not sure If I get right what cases should be tested

Reply:
Never mind, nothing to do here
I was thought about some checking, that it will work even with not default pool, but looks like it will not be the case here and should be tested separately with introducing some leak tracing mechanism